rtrlib/rtr/rtr.c

```diff
@@ -52,6 +53,127 @@ static int install_sig_handler()
     return sigaction(SIGUSR1, &sa, NULL);
 }
 
+int get_interval_mode()
+{
+    return interval_option;
+}
+
+void set_interval_mode(int option)
+{
+    const char* txt;
+    switch(option) {
+    case IGNORE_ANY:
+        interval_option = option;
+        txt = "IGNORE_ANY";
+        break;
+    case ACCEPT_ANY:
+        interval_option = option;
+        txt = "ACCEPT_ANY";
+        break;
+    case DEFAULT_MIN_MAX:
+        interval_option = option;
+        txt = "DEFAULT_MIN_MAX";
+        break;
+    case IGNORE_ON_FAILURE:
+        interval_option = option;
+        txt = "IGNORE_ON_FAILURE";
+        break;
+    default:
+        RTR_DBG1("Invalid interval mode. Mode remains unchanged.");
+        return;
+    }
+    RTR_DBG("Interval mode set to %s", txt);
```

Comment:
This functions complexity is only related to this debug output, otherwise it should be just:
```
void set_interval_mode(int option)
{
    switch(option) {
    case IGNORE_ANY:
    case ACCEPT_ANY:
    case DEFAULT_MIN_MAX:
    case IGNORE_ON_FAILURE:
        interval_option = option;
        break;
    default:
         RTR_DBG1("Invalid interval mode. Mode remains unchanged.");
    }
}
```